src/components/panes/EditCampaignPane.jsx

```diff
@@ -41,6 +42,17 @@ export default class EditCampaignPane extends PaneBase {
         }
     }
 
+    getPaneSubTitle(data) {
+        if (this.props.campaignItem && !this.props.campaignItem.isPending) {
+            const link= 'https://zetk.in/o/'+this.props.activeOrg.id+'/campains/'+this.props.campaignItem.data.id;
```

Comment:
This hardcodes the URL to the domain `zetk.in` using HTTPS, which is correct for the live environment but not correct for the development environment. Instead of https, use `//` (same scheme as current page) and instead of the domain, use `process.env.ZETKIN_DOMAIN`.

See _Dashboard.jsx_ for a similar solution:
https://github.com/zetkin/organize.zetk.in/blob/feacfdfba5d2f08c39b6c03e414e90d10c2d39f6/src/components/dashboard/Dashboard.jsx#L65